examples/segnet/train.py

```diff
@@ -0,0 +1,104 @@
+import matplotlib  # isort:skip # NOQA
+matplotlib.use('Agg')  # isort:skiip # NOQA
+
+import argparse
+import time
+
+import chainer
+import numpy as np
+
+from chainer import iterators
+from chainer import optimizers
+from chainer import training
+from chainer.training import extensions
+from chainercv.datasets import CamVidDataset
+from chainercv.datasets import TransformDataset
+from chainercv.links.loss import PixelwiseSoftmaxLossWithWeight
+from chainercv.links.model.segnet import segnet_basic
+
+
```

Comment:
How about adding an option for output directory and writing scripts in `main`.
I think other options like `resume` are not necessary.